<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manipulation</class>
 <widget class="QDialog" name="Manipulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>426</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Manipulation</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>311</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>File Order</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>To change the order of imported/appended files, 
please re-arrange the list below. 
Drag and drop individual files for a desired order.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="listFileNames">
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>181</width>
     <height>47</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Sampling Frequency:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSpinBox" name="lineEditSampleFreq">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000000</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="checkSubsampling">
      <property name="text">
       <string>Downsample on next import</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupDownample">
      <property name="title">
       <string>Downsample Settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>New sampling frequency:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSpinBox" name="lineEditNewSampleFreq">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10000000</number>
             </property>
             <property name="value">
              <number>200</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Hz</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Manipulation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Manipulation</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkSubsampling</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupDownample</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
